baseplate/clients/sqlalchemy.py

```diff
@@ -209,14 +209,6 @@ def __init__(self, engine: Engine, name: str = "sqlalchemy"):
         event.listen(self.engine, "handle_error", self.on_error)
         self.time_started = 0.0
 
-        # Prometheus pool metrics
-        pool = self.engine.pool
-        if isinstance(pool, QueuePool):
-            self.max_connections_gauge.labels(name).set_function(pool.size)
-            self.checked_out_connections_gauge.labels(name).set_function(
-                pool.checkedout + pool.overflow
-            )
-
```

Comment:
You may want to reintroduce these in report_runtime_metrics, they're not called atm

Reply:
They are there already :)

Reply:
oh they are, and checkedout _includes_ overflow, nevermind